src/test/java/org/folio/utility/LocationUtility.java

```diff
@@ -0,0 +1,242 @@
+package org.folio.utility;
+
+import static org.folio.rest.api.TestBase.get;
+import static org.folio.rest.support.http.InterfaceUrls.locCampusStorageUrl;
+import static org.folio.rest.support.http.InterfaceUrls.locInstitutionStorageUrl;
+import static org.folio.rest.support.http.InterfaceUrls.locLibraryStorageUrl;
+import static org.folio.rest.support.http.InterfaceUrls.locationsStorageUrl;
+import static org.folio.rest.support.http.InterfaceUrls.servicePointsUrl;
+import static org.folio.utility.RestUtility.send;
+
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import java.net.MalformedURLException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.folio.rest.jaxrs.model.HoldShelfExpiryPeriod;
+import org.folio.rest.jaxrs.model.StaffSlip;
+import org.folio.rest.support.Response;
+import org.folio.rest.support.ResponseHandler;
+
+public class LocationUtility {
+  private static final String SUPPORTED_CONTENT_TYPE_JSON_DEF = "application/json";
+  private static UUID institutionID;
+  private static UUID campusID;
+  private static UUID libraryID;
+  private static final List<UUID> servicePointIDs = new ArrayList<>();
+
+  private LocationUtility() {
+    throw new UnsupportedOperationException("Cannot instantiate utility class.");
+  }
+
+  public static void createLocationUnits(boolean force) {
+
+    try {
+      if (force || institutionID == null) {
+        institutionID = UUID.randomUUID();
+        createInstitution(institutionID, "Primary Institution", "PI");
+
+        campusID = UUID.randomUUID();
+        createCampus(campusID, "Central Campus", "CC", institutionID);
+
+        libraryID = UUID.randomUUID();
+        createLibrary(libraryID, "Main Library", "ML", campusID);
+
+        UUID spID = UUID.randomUUID();
+        servicePointIDs.add(spID);
+        String spName = "Service Point " + spID.toString();
+        createServicePoint(spID, spName, "SP" + spID.toString(), spName, "SP Description", 0, false, null);
+      }
+    } catch (Exception e) { // should not happen
+      throw new AssertionError("CreateLocUnits failed:", e);
+    }
```

Comment:
Could sneaky throws be used to avoid this?

Reply:
I left a lot of the original code or logic alone as much as possible when moving things into the utility.
The problem is that there is no obvious change set to compare against.

I have no problems changing this.
The try..catch.. isn't even needed here.
I can get away without even using sneakythrows.

I'm going to remove the try..catch.. but I am not add the `@SneakyThrows` (unless you explicitly want `@SneakyThrows` added).

Reply:
I only suggested the SneakyThrows because I assumed that the catch related to an expected exception. 

If there isn't, then I have no need for SneakyThrows